source/deploy/server/preparations.rst

```diff
@@ -0,0 +1,183 @@
+
+      .. code-block:: sql
+
+         GRANT ALL PRIVILEGES ON DATABASE mattermost to mmuser;
+
+   e. If using PostgreSQL v15.x or later, additional grants are required:
+
+      .. code-block:: sql
+
+         ALTER DATABASE mattermost OWNER TO mmuser;
+         GRANT USAGE, CREATE ON SCHEMA PUBLIC TO mmuser;
+
+3. Configure PostgreSQL for remote connections (if database is on a separate server):
+
+   a. Edit postgresql.conf to allow remote connections:
+      
+      .. tab:: Ubuntu/Debian
+      
+         Edit ``/etc/postgresql/{version}/main/postgresql.conf``:
+         
+         .. code-block:: text
+
+            listen_addresses = '*'
+
+      .. tab:: Red Hat/CentOS
+      
+         Edit ``/var/lib/pgsql/{version}/data/postgresql.conf``:
+         
+         .. code-block:: text
+
+            listen_addresses = '*'
+
+   b. Configure client authentication by editing pg_hba.conf:
+      
+      Add the following line, replacing {mattermost-server-IP}:
+      
+      .. code-block:: text
+
+         host all all {mattermost-server-IP}/32 md5
+
+File storage preparation
+-------------------------
+
+Mattermost requires a file storage system for storing user files, images, and attachments. You have several options, including:
+
+- local file storage
+- S3-compatibile object storage (recommended)
+
+Local file storage
+~~~~~~~~~~~~~~~~~~
+
+For simple deployments, you can use local file storage. However, this is not recommended for production environments or multi-node deployments.
+
+1. Create a directory for file storage:
+
+   .. code-block:: sh
+
+      sudo mkdir -p /opt/mattermost/data
+
+2. Set appropriate permissions:
+
+   .. code-block:: sh
+
+      sudo chown -R mattermost:mattermost /opt/mattermost/data
+
+S3-compatible object storage (Recommended)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For production environments, we recommend using S3-compatible object storage such as:
+
+- Amazon S3
+- MinIO
+- Digital Ocean Spaces
+- Other S3-compatible services
+
+When using S3 storage, you'll need:
+
+1. A bucket created specifically for Mattermost
+2. Access credentials (Access Key and Secret Key)
+3. Appropriate bucket policies configured
+4. The following information for configuration:
+   - Bucket name
+   - Region (if applicable)
+   - Access Key
+   - Secret Key
+   - Endpoint URL (for non-AWS S3 services)
+
+Network preparation
+--------------------
+
+Ensure the following ports are available:
+
+- Application ports: 80/443 (TCP) for HTTP/HTTPS
+- Database port: 5432 (TCP) for PostgreSQL
+- SMTP port: 10025 (TCP/UDP) for outbound email
+
+System requirements
+--------------------
+
+Ensure your system meets these minimum requirements:
+
+- Operating System: 64-bit Linux distribution
+- Hardware: 1 vCPU/core with 2GB RAM (supports up to 1,000 users)
+- Storage: Minimum 10GB available space
+- Database: PostgreSQL v13+
+- Network: Reliable internet connection with sufficient bandwidth
```

Comment:
Do we have more detailed guidelines on the system requirements in the new docs? In the old version, there was a handy page with the suggested resources (https://docs.mattermost.com/scale/scaling-for-enterprise.html). Does that still exist?